dynatrace-activegate-operator-google-marketplace/charts/dynatrace-activegate-operator/values.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+operator:
+  image: ""
+
+activegate:
+  name: "activegate"
+  apiUrl: ""
+  image: ""
+  disableActivegateUpdate: false
```

Comment:
```suggestion
  disableActiveGateUpdate: false
```

Reply:
It's like that - if we want to change it we also need to make the change in all corresponding places
(see https://github.com/Dynatrace/dynatrace-activegate-operator/blob/master/pkg/apis/dynatrace/v1alpha1/activegate_types.go#L27)

Reply:
Let's wait until the Operator/CRD names are decided, but in general, it's 'ActiveGate'.